com.github.emmanueltouzery.cigale.json

```
@@ -0,0 +1,49 @@
+{
+  "app-id": "com.github.emmanueltouzery.cigale",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "cigale",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
+    "--share=network"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "env": {
+      "CARGO_HOME": "/run/build/rust-flatpak/cargo"
```

```suggestion
      "CARGO_HOME": "/run/build/cigale/cargo"
```
With this changed, you can remove the cargo-config dir as it's not needed.

---

I don't fully understand. I'll change that line and check if everything still works afterwards

---

I'm not sure what you don't understand? that line sets the CARGO HOME to /run/build/$module-name/cargo. In your case, the module name is cigale. Your vendored files can then be found and everything should work without including a custom cargo config file.